src/latexify/codegen/function_codegen.py

```diff
@@ -536,6 +536,30 @@ def visit_IfExp(self, node: ast.IfExp) -> str:
         latex += self.visit(node)
         return latex + r", & \mathrm{otherwise} \end{array} \right."
 
+    def visit_Match(self, node: ast.Match) -> str:
+        """Visit a match node"""
+        latex = r"\left\{ \begin{array}{ll} "
+        subject_latex = self.visit(node.subject)        
+        for match_case in node.cases:
```

Comment:
In the current implementation of this library, we don't allow fall-through syntax, and the statement must return any value. In the case of match, it should have either:

- Cases that cover every possible conditions
- Cases with wildcards that are converted to `otherwise`

This is necessary to express the correct math expressions, and we need to check if the given code is correctly expressing this syntax.

Since the first option is hard to check, the second option is the only approach we could have: deny all `Match` subtrees without wildcards.